evalml/pipelines/components/transformers/preprocessing/transform_primitive_components.py

```diff
@@ -72,20 +72,13 @@ def transform(self, X, y=None):
 
         es = self._make_entity_set(X_ww)
         features = ft.calculate_feature_matrix(features=self._features, entityset=es)
-
         features.set_index(X_ww.index, inplace=True)
 
         X_ww = X_ww.ww.drop(self._columns)
+        features.ww.init(logical_types={col_: "categorical" for col_ in features})
```

Comment:
So you're taking all features generated from DFS and Calculate Feature Matrix, and making them categorical?

Reply:
Yes but currently this only applies for URL and Email features. It was the old behavior but now the problem is that ww 0.6.1 infers features created by ft as Unknown.